front/src/styles/scss/applications/stdcm/_loader.scss

```
@@ -1,61 +1,102 @@
-.stdcm-loader-background {
-  padding: 0 0 48px 338px;
-  background-color: #eff3f5;
-}
 .stdcm-loader {
-  padding: 48px 48px 19px 48px;
-  border-radius: 6px;
-  background-color: rgba(255, 255, 255, 1);
-  width: 546px;
+  z-index: 2;
+  padding: 16px 16px 18px;
```

I think we can do this
```suggestion
  padding: 16px 18px;
```

---

We can't. `padding: 16px 16px 18px;` stands for 16 top, 16 inline (left and right) and 18 bottom. This won't be the same behavior with `padding: 16px 18px;`

---

My bad !